[Cpp Runtime] MSVC: ANTLRFileStream uses removed "s2ws"

Hello,

master branch of runtime cpp fails to compile with MSVC on the following line:

![image](https://user-images.githubusercontent.com/667557/148354336-9a5bbedc-6bf0-4e07-9144-a9e49d708c1b.png)

thanks for your support,

@jcking Release 4.10 still fails to compile. Runtime source does not provide any definition for `antlrcpp::s2ws`:

```
antlr4-cpp-runtime\ANTLRFileStream.cpp(17): error C2653: 'antlrcpp': is not a class or namespace name
antlr4-cpp-runtime\ANTLRFileStream.cpp(17): error C3861: 's2ws': identifier not found
```
From my attempt to upgrade to 4.10: https://github.com/SystemRDL/systemrdl-compiler/runs/6001839774

--- a/runtime/Cpp/runtime/src/ANTLRFileStream.cpp
+++ b/runtime/Cpp/runtime/src/ANTLRFileStream.cpp
@@ -13,11 +13,7 @@ void ANTLRFileStream::loadFromFile(const std::string &fileName) {
     return;
   }
 
-#ifdef _MSC_VER
-  std::ifstream stream(antlrcpp::s2ws(fileName), std::ios::binary);
-#else
   std::ifstream stream(fileName, std::ios::binary);
-#endif
 
   ANTLRInputStream::load(stream);
 }

